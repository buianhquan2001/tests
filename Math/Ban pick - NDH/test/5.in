10
275 0.5296
285 0.6982
265 0.6248
253 0.5069
294 0.0679
260 0.6447
255 0.9401
287 0.3884
299 0.4018
270 0.3851
