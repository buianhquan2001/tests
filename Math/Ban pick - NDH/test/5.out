102
146
119
90
14
121
202
77
83
71
